killrweather-clients/src/main/scala/com/datastax/killrweather/DataFeedApp.scala

```diff
@@ -122,7 +117,7 @@ class DynamicDataFeedActor(cluster: Cluster) extends Actor with ActorLogging wit
     case HttpRequest(POST, Uri.Path(ctx), headers, entity, _) =>
       toFiles(headers) map { files =>
         log.info(s"Received {}", files.mkString)
-        context.actorOf(Props(new FileFeedActor(cluster))) ! FileStreamEnvelope(files:_*)
+        context.actorOf(Props(new AutomaticDataFeedActor(cluster))) ! FileStreamEnvelope(files:_*)
```

Comment:
This is incorrect. The automatic data feed actor's job is to automate. The feed actor's job is to handle dynamic submissions, i.e. via the REST api.